modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -82,6 +82,7 @@ def request_decision
       def upload_document
         jid = if Flipper.enabled?(:mobile_lighthouse_document_upload, @current_user)
                 params[:claim_id] = params[:id]
+                params[:tracked_item_ids] = Array.wrap(params[:tracked_item_id])
```

Comment:
One small thing I'll point out is that we're passing extra, unused fields to the upstream service. I'm sure it just ignores them, but I think alternatively we could probably do something like 
`params[:tracked_item_ids] = Array.wrap(params.delete(:tracked_item_id))`
which would prevent us from also sending `tracked_item_id`

Reply:
good point. updated to delete the parameter. Also found a way to test for the tracked item ids so added that.